testing/test_assertrewrite.py

```diff
@@ -173,10 +190,16 @@ def __init__(self):
         def f():
             assert cls().foo == 2  # noqa
 
-        message = getmsg(f, {"cls": Y})
-        assert "assert 3 == 2" in message
-        assert "+  where 3 = Y.foo" in message
-        assert "+    where Y = cls()" in message
+        # XXX: looks like the "where" should also be there in verbose mode?!
+        message = getmsg(f, {"cls": Y}).splitlines()
+        if request.config.getoption("verbose"):
+            assert message == ["assert 3 == 2", "  -3", "  +2"]
+        else:
+            assert message == [
+                "assert 3 == 2",
+                " +  where 3 = Y.foo",
+                " +    where Y = cls()",
+            ]
```

Comment:
This is not clear to me (see the "XXX") - an uncovered bug?

Reply:
@asottile 
Any idea?  (since it is about assertion rewriting)

Reply:
that is quite weird!

Looking at the rich compare it looks like the verbose diff mode skips the `where` metalanguage entirely: https://github.com/pytest-dev/pytest/blob/77643122a8d090f951d227ccc7c672deb4e8466c/src/_pytest/assertion/util.py#L154-L155

Reply:
=> https://github.com/pytest-dev/pytest/issues/5192